docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -196,7 +234,8 @@ Assumptions and Limitations
 
 - We assume that CPAP availability captures actual use, and not simply the machine being in the facility 
 - We assume that the delivery facility is the final facility in the care continum for deliveries that are transferred due to complications
-- We assume that the relative risk of RDS mortality with CPAP in practice is similar to that found in the Cochrane Review meta-analysis
+- We assume that the relative risk of RDS mortality with CPAP in practice is similar to that found in the Cochrane Review meta-analysis. Given that the review assessed overall mortality rather than RDS mortality, it is likely that we will underestimate the overall impact of CPAP on mortality in our simulation.
```

Comment:
This is interesting. I guess we would need to back calculate an impact on RDS or at least pre-term that got the effect on overall mortality found here? 

I made a Jira ticket for this data analysis and update - feel free to adjust the description. I put it in model expansion high priority. If we want to improve this, I think it would be worthwhile. Ticket: https://jira.ihme.washington.edu/browse/SSCI-2286